index.html

```diff
@@ -0,0 +1,177 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,900&display=swap');
+        </style>
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+
+<link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap');
+        </style>
+    <title>Visit Rwanda Festivals and Events</title>
+</head>
+<body>
+    <div class="social_icon">
+        <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+        </ul>
+    </div>
+    <header id="header">
+        <div class="logo">
+          <a href="index.html">
+          <img src="images/gorilla-logo.png" alt=""></a>
+        </div>
+        <nav id="nav-bar" class="nav-bar d-none">
+          
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="aboutpage.html" >About</a>
+            <a class="nav-link visited" href="#programs" >Program</a>
+            <a class="nav-link" href="#contents" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#sponsor" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+          
+        </div>
+        <div id="close-icon" class="close-icon d-none">
+            <i class="fas fa-times-circle"></i>
+          </div>
+      </header>
+    <div class="banner" >
+        <div class="background">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals <br> and Events</h2>
+        <p>If you’re looking to get out of the big city and back into nature, the annual Kwita Izina gorilla-naming ceremony is a special time to visit Volcanoes National Park. The annual naming ceremony takes place in Kinigi,</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+    </div>
+    </div>
+    <main id="programs">
```

Comment:
- [ ] The items within the _main program_ section of your design do not follow the typography (**font family**, **font-weight**, ...), and have appropriate **padding** and **margins** around the items, as well as **spacing** between the contents of each item. Kindly ensure that there's enough padding around the item's contents.

**Your program section:**
![image](https://user-images.githubusercontent.com/12745474/133249578-29fd5b2b-dc88-49e6-8ac9-f361310dbbe5.png)

**The original program section:**
![image](https://user-images.githubusercontent.com/12745474/132545825-150c3b77-7c05-4a8c-aa95-956269677499.png)

Reply:
- [ ] As said by the previous code reviewer, try as much as possible to respect the original design of the project. The spacing between the icons and the titles is too much and the font family is not respected. Please kindly fix that.